schema/resolvers.js

```diff
@@ -171,7 +173,26 @@ const Resolvers = {
     updateMetadata: (_, args, ctx) =>
       ctx.repositories.Metadata.update(args.input),
     deleteMetadata: (_, args, ctx) =>
-      ctx.repositories.Metadata.remove(args.input.id)
+      ctx.repositories.Metadata.remove(args.input.id),
+
+    createQuestionConfirmation: (_, args, ctx) =>
+      ctx.repositories.QuestionConfirmation.create(args.input),
+    updateQuestionConfirmation: (
+      _,
+      { input: { positive, negative, ...rest } },
+      ctx
+    ) =>
+      ctx.repositories.QuestionConfirmation.update({
+        ...rest,
```

Comment:
Is there a need to collect the rest of the parameters? I _think_ you just need `title`?